src/main/java/mate/academy/hibernate/relations/service/impl/CountryServiceImpl.java

```diff
@@ -1,16 +1,23 @@
 package mate.academy.hibernate.relations.service.impl;
 
+import mate.academy.hibernate.relations.dao.CountryDao;
 import mate.academy.hibernate.relations.model.Country;
 import mate.academy.hibernate.relations.service.CountryService;
 
 public class CountryServiceImpl implements CountryService {
+    private final CountryDao countryDao;
+
+    public CountryServiceImpl(CountryDao countryDao) {
+        this.countryDao = countryDao;
+    }
+
     @Override
     public Country add(Country country) {
-        return null;
+        return countryDao.add(country);
     }
 
     @Override
     public Country get(Long id) {
-        return null;
+        return countryDao.get(id).orElseThrow();
```

Comment:
The `get` method uses `orElseThrow()` without specifying an exception. This will throw a `NoSuchElementException` by default if the country is not found. Consider specifying a more descriptive exception to provide better context for the error.